src/test/groovy/com/datadog/ddwaf/exception/AbstractWafExceptionTest.groovy

```diff
@@ -40,11 +39,17 @@ class AbstractWafExceptionTest {
          * This makes it perfect for testing the unhandled enum case without risking interference
          * with actual WAF error scenarios.
          */
```

Comment:
I doubt this test or this code should change in this PR. Let's restor ethe previous version if possible.